### ISSUE TITLE
J::V 4.10+ depends on Mojo 8

### ISSUE THREAD

**Issue by 1nickt** [mojolicious/json-validator#235]
* JSON::Validator version: 4.10+
* Perl version: 5.30+
* Operating system: Darwin
* Mojolicious version : 7.94

### Steps to reproduce the behavior
J::V states Mojolicious dependency of 7.2 but it uses the `raise` cmd which is in Mojo 8.  This can be seen easily by installing Mojo 7.94 and hen trying to install JSON::Validator 4.10 or above.

### Actual behavior
Running the app crashes immediately because the `raise` bareword is not known. Same as when the installation tries to test:
```
#   Failed test 'use JSON::Validator'
#   at t/00-basic.t line 34.
# Can't locate raise.pm in @INC (you may need to install the raise module)
```

**Comment by jhthorsen** [comment]
Should be fixed in 4.12.

**Comment by 1nickt** [comment]
Thanks you for the quick fix @jhthorsen
